src/react/snippet.tsx

```diff
@@ -9,9 +9,7 @@ import { partytownSnippet } from '@builder.io/partytown/integration';
  *
  * @public
  */
-export interface PartytownProps extends PartytownConfig {
-  nonce?: string;
-}
+export interface PartytownProps extends PartytownConfig {}
```

Comment:
why are we removing this property here?

Reply:
This was added recently in one of the commits, but this covered only React configuration. Since nonce is declared on the global config, there’s no need to keep the property in this interface.